src/ctia/http/middleware/auth.clj

```diff
@@ -15,22 +16,23 @@
         (assoc-in [:headers "authorization"] auth-header))
     request))
 
-(defn testable-wrap-authentication
+(s/defn testable-wrap-authentication
   "wrap-autentication middleware."
-  [handler auth-service]
+  [handler
+   identity-for-token :- (s/=> s/Any s/Any)]
```

Comment:
> "I learnt a few days ago that s/=> has no runtime semantics
very sad
at least according to the docs
it's actually just an ifn? check haha"

[Ambrose  2020] :-)
```suggestion
   identity-for-token :- (s/pred ifn?)]
```

Reply:
I was just stating a fact that seemed to be misunderstood by the team, I was surprised to see this interpreted as "use ifn? over s/=>". I feel the opposite :)